docs/sheets/GNSS.md

```diff
@@ -1,236 +1,251 @@
 # COMMON GNSS TASKS
 
-* :scroll: :one: [Creating or updating a GNSS __mark__](#mark)
-* :scroll: :two: [Creating or updating a GNSS __monument__](#monument)
-* :scroll: :three: [Installing or updating a GNSS __antenna__](#antenna)
-* :scroll: :four: [Installing or updating a GNSS __receiver__](#receiver)
-* :scroll: :five: [Installing or updating a GNSS __receiver__ firmware version](#firmware)
-* :scroll: :six: [Installing or updating a GNSS __receiver__ session configuration](#session) :dragon:
 * :hammer: [Combined editing steps](#steps)
+* :scroll: :one: [GNSS __receiver firmware version__ upgrade](#firmware)
+* :scroll: :two: [GNSS __receiver__ change](#receiver) :dragon: 
+* :scroll: :three: [GNSS __antenna__ change](#antenna) :dragon: 
+* :scroll: :four: [GNSS __receiver settings__ change](#session) :dragon:
+* :scroll: :five: [GNSS __coordinates__ update](#coordinates)
+* :scroll: :six: [__creating updating or closing__ a GNSS site](#openclose) :dragon:
+
 
 Global Navigation Satellite System (GNSS) equipment are installed at GNSS __marks__. Each __mark__ is
 attached to a __monument__. The details of the __mark__, as found in the __network/marks.csv__ file,
-tend to relate more to documentation, such as name and location. Whereas the __network/monuments.csv__ file
+tend to relate more to documentation, such as mark identification and location. Whereas the __network/monuments.csv__ file
 tends to list physical aspects of the actual monument as constructed.
 
 Equipment installed should be found in the the relevant __assets__ directory files. Actual times when
 __antennas__ and __receivers__ are deployed is managed via the __install/antennas.csv__ and __install/receiver.csv__
 files respectively. As the firmware of the __receiver__ is important for data post-processing, this is maintained
 in the __install/firmware.csv__ file. This is independent of installation and can also be maintained for instruments
-not actively deployed.
+not actively deployed. __radomes__ and __metsensors__ are occasionally installed at GNSS marks.
 
 Describing how the data is being recorded is managed through the __install/sessions__ file. This correlates with the general model
 of the receiver being deployed and the __session__ sample "intervals". Other meta-data information required for post-processing
 or analysis is also stored in this file. These settings can be delimited by time ranges and are not dependent on
 any particular installed receiver but only on the receiver model type.
 
-## :one: <a name="mark"></a>_Creating or updating a GNSS mark_
+A complete list of files that cover GNSS metadata is below.
 
-For new marks a __code__ will need to be assigned by the appropriate mechanism.
+folder | tables
+-------|-------
+assets | antennas.csv metsensors.csv radomes.csv receivers.csv
+environment | visibility.csv
+install | antennas.csv firmware.csv metsensors.csv radomes.csv receivers.csv sessions.csv
+network | marks.csv monuments.csv networks.csv
```

Comment:
this is a little difficult to read, could we add delimiters such as , or | in the tables column?